.pre-commit-config.yaml

```diff
@@ -1,7 +1,9 @@
 repos:
-  - repo: https://github.com/python/black
-    rev: 19.3b0
-    hooks:
-      - id: black
-        language_version: python3.7
-        exclude_types: ['markdown', 'ini', 'toml']
+- repo: local
```

Comment:
could you explain a bit what the new changes are?

Reply:
The black's pre-commit-config.yaml is taken from https://github.com/psf/black/blob/master/.pre-commit-config.yaml